h/streamer/conf/newrelic.ini

```diff
@@ -0,0 +1,332 @@
+transaction_tracer.explain_enabled = false
+
+# Threshold for query execution time below which query plans
+# will not not be captured. Relevant only when "explain_enabled"
+# is true.
+transaction_tracer.explain_threshold = 0.5
+
+# Space separated list of function or method names in form
+# 'module:function' or 'module:class.function' for which
+# additional function timing instrumentation will be added.
+transaction_tracer.function_trace =
+
+# The error collector captures information about uncaught
+# exceptions or logged exceptions and sends them to UI for
+# viewing. The error collector is enabled by default. Set this
+# to "false" to turn it off.
+error_collector.enabled = false
+
+# To stop specific errors from reporting to the UI, set this to
+# a space separated list of the Python exception type names to
+# ignore. The exception name should be of the form 'module:class'.
+error_collector.ignore_errors =
+
+# Browser monitoring is the Real User Monitoring feature of the UI.
+# For those Python web frameworks that are supported, this
+# setting enables the auto-insertion of the browser monitoring
+# JavaScript fragments.
+browser_monitoring.auto_instrument = false
+
+# A thread profiling session can be scheduled via the UI when
+# this option is enabled. The thread profiler will periodically
+# capture a snapshot of the call stack for each active thread in
+# the application to construct a statistically representative
+# call tree.
+thread_profiler.enabled = false
+
+# Your application deployments can be recorded through the
+# New Relic REST API. To use this feature provide your API key
+# below then use the `newrelic-admin record-deploy` command.
+# api_key =
+
+# Distributed tracing lets you see the path that a request takes
+# through your distributed system. Enabling distributed tracing
+# changes the behavior of some New Relic features, so carefully
+# consult the transition guide before you enable this feature:
+# https://docs.newrelic.com/docs/transition-guide-distributed-tracing
+distributed_tracing.enabled = false
+
+# Turn off everythign we can find. For details see:
+# https://docs.newrelic.com/docs/agents/python-agent/configuration/python-agent-configuration
+
+disable_all_plugins = true
+
+transaction_events.enabled = false
+transaction_segments.attributes.enabled = false
+browser_monitoring.enabled = false
+datastore_tracer.instance_reporting.enabled = false
+custom_insights_events.enabled = false
+span_events.enabled = false
+event_loop_visibility.enabled = false
+cross_application_tracer.enabled = false
+slow_sql.enabled = false
+
+# Disabling various detection settings
+utilization.detect_aws = false
+utilization.detect_azure = false
+utilization.detect_gcp = false
+utilization.detect_pcf = false
+utilization.detect_docker = false
+
+# Disable various instrumentations of modules
+# These can be discovered by setting the "log_level" to "debug" and
+# observing "instrument module" messages on start up
+
```

Comment:
This is very fun. The documentation handily tells you that the agent instruments things as they are loaded, but you can turn them off if you like, but finding every one of them and disabling them individually.